components/application-mgt/org.wso2.carbon.identity.application.mgt/src/main/java/org/wso2/carbon/identity/application/mgt/ApplicationManagementService.java

```diff
@@ -131,6 +131,14 @@ public abstract void updateApplication(ServiceProvider serviceProvider, String t
     public abstract void deleteApplication(String applicationName, String tenantDomain, String username)
             throws IdentityApplicationManagementException;
 
+    /**
+     * Delete Applications by tenant id.
+     *
+     * @param tenantId Id of the tenant
+     * @throws IdentityApplicationManagementException
```

Comment:
Complete the Javadoc comment by adding the specific Exception type. The param descriptions should start with a capital letter and end with a full stop.

Reply:
Check every other instance and fix this

Reply:
Fixed in #3312